src/codewhisperer/models/constants.ts

```diff
@@ -122,8 +122,15 @@ export const licenseFilter = 'CodeWhisperer suggestions were filtered due to ref
 export const welcomeCodeWhispererReadmeFileSource = 'resources/markdown/WelcomeToCodeWhisperer.md'
 
 export const welcomeCodeWhispererCloud9Readme = 'resources/markdown/WelcomeToCodeWhispererCloud9.md'
+/**
+ * oldWelcomeMessageKey is a key for the previous welcome page of version v1.91.0,
+ */
+export const oldWelcomeMessageKey = 'CODEWHISPERER_WELCOME_MESSAGE'
 
-export const welcomeMessageKey = 'CODEWHISPERER_WELCOME_MESSAGE'
+/**
+ *  newWelcomeMessageKey is a new key for the new getting started experience, it is used to track if first time signIn new user/ existing user has seen the getting started page or not.
+ */
+export const newWelcomeMessageKey = 'CODEWHISPERER_WELCOME_MESSAGE_SHOWN'
```

Comment:
not a blocker but the actual string can be better to indicate it's key for the new UX.

Reply:
We are not using this Keys anymore as we are showing the Banner for both new and old users all the time.

Reply:
just to confirm, we no longer need the `oldWelcomeMessageKey`, the new one is still needed.

Reply:
As we are using PromptSettings we are not using these keys.